tailwind.config.js

```diff
@@ -52,34 +52,36 @@ module.exports = {
       DEFAULT: "4px",
     },
     colors: {
-      black: "#000000",
-      white: "#ffffff",
-      nightshade: "#1D0023",
-      eggplant: "#17063b",
-      "dark-blurple": "#2f0c7a",
-      "main-blurple": "#563acc",
-      "accent-blurple": "#6364ff",
-      "blurple-text-on-black": "#858afa",
-      lime: "#baff3b",
-      goldenrod: "#ffbe2e",
-      "gray-1": "#555555",
-      "gray-2": "#9b9b9b",
-      "gray-3": "#d4d4d4",
-      "gray-4": "#f3f3f3",
-      "gray-5": "#f6f6f6",
+      black: "var(--black)",
+      white: "var(--white)",
+      nightshade: "var(--nightshade)",
+      eggplant: "var(--eggplant)",
+      "dark-blurple": "var(--dark-blurple)",
+      "main-blurple": "var(--main-blurple)",
+      "accent-blurple": "var(--accent-blurple)",
+      "blurple-text-on-black": "var(--blurple-text-on-black)",
+      lime: "var(--lime)",
+      goldenrod: "var(--goldenrod)",
+      "gray-1": "var(--gray-1)",
+      "gray-2": "var(--gray-2)",
+      "gray-3": "var(--gray-3)",
+      "gray-4": "var(--gray-4)",
+      "gray-5": "var(--gray-5)",
```

Comment:
moved these colors to css variables so we're not writing the hex codes twice. Alternatively, we could just define the colors in the tailwind config but it's kinda nice to be able to reference these from the dev tools :~)